File: app/src/main/java/com/mehmetakiftutuncu/muezzin/activities/preferences/ReminderPreferencesActivity.java
package com.mehmetakiftutuncu.muezzin.activities.preferences;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.mehmetakiftutuncu.muezzin.R;
import com.mehmetakiftutuncu.muezzin.fragments.preferences.ReminderPreferencesFragment;

public class ReminderPreferencesActivity extends AppCompatActivity {
    @Override protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_reminderpreferences);

        ReminderPreferencesFragment reminderPreferencesFragment = new ReminderPreferencesFragment();

        getFragmentManager()
                .beginTransaction()
                .replace(R.id.frameLayout_reminderPreferencesContainer, reminderPreferencesFragment)
                .commit();
    }
}

File: app/src/main/java/com/mehmetakiftutuncu/muezzin/activities/preferences/ReminderPreferencesActivity.kt
package com.mehmetakiftutuncu.muezzin.activities.preferences

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.mehmetakiftutuncu.muezzin.R
import com.mehmetakiftutuncu.muezzin.fragments.preferences.ReminderPreferencesFragment

class ReminderPreferencesActivity: AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_reminderpreferences)

        supportFragmentManager
                .beginTransaction()
                .replace(R.id.frameLayout_reminderPreferencesContainer, ReminderPreferencesFragment())
                .commit()
    }
}